src/services/requestHandler.js

```diff
@@ -0,0 +1,73 @@
+import axios from 'axios';
+
+/**
+ * This function creates an instance of axios and return
+ * it for making API request
+ * @param { Object} headers an object of http headers
+ * @param { Object } body the request body
+ * @returns { instance } an instance of axios
+ */
+const axiosInstance = (headers, body) => {
+  let apiUrl = 'https://authorhaven.herokuapp.com/api';
+  if (process.env.NODE_ENV === 'development') {
+    apiUrl = 'http://localhost:3000/api';
+  }
+  const instanceCreate = axios.create({
+    baseURL: apiUrl,
+    headers,
+    body,
+  });
+  return instanceCreate;
+};
+
+/**
+ * This function can be used for making http request
+ * passing in an instance of axios and other desire params
+ * @param {*} axiosInStance an instance of axios
+ * @param { String } method http method e.g post, get, put, delete
+ * @param { String } path the relative path of the request
+ * @param { Object } headers an object http headers
+ * @param { Object } body the request body
+ * required for making post and put request
+ * @returns { Object } response the response from the server
+ */
+const apiCalls = async (axiosInStance, method, path, headers, body) => {
+  /* eslint-disable no-console */
```

Comment:
take all this out, you don't need it, it just complicates things

Reply:
Ok, I will attend to them.